gatsby-site/.gitignore

```diff
@@ -2,6 +2,7 @@
 .cache
 node_modules
 yarn-error.log
+package-lock.json
```

Comment:
Ellie told me we should keep the package-lock after all!  I'm going to remove this from my gitignore.

Reply:
Awesome - I took it out of mine too.